src/Renci.SshNet.Tests/Classes/ServiceFactoryTest_CreateSftpFileReader_FileSizeIsMoreThanMaxPendingReadsTimesChunkSize.cs

```diff
@@ -22,18 +22,20 @@ public class ServiceFactoryTest_CreateSftpFileReader_FileSizeIsMoreThanTenTimesG
         private SftpFileAttributes _fileAttributes;
         private long _fileSize;
         private ISftpFileReader _actual;
+        private int _maxPendingReads;
 
         private void SetupData()
         {
             var random = new Random();
 
+            _maxPendingReads = 100;
```

Comment:
This value needs to be the same as ServiceFactory.cs:145
I've renamed this testcase to match what it does